src/plugins/data_views/server/routes/has_data_views.ts

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { IRouter } from '../../../../core/server';
+import { hasIndexPattern, hasUserIndexPattern } from '../has_user_index_pattern';
+
+export const registerHasDataViewsRoute = (router: IRouter): void => {
```

Comment:
I'm tempted to avoid creating a REST API for this as it effectively means we can't make breaking changes to this for a minimum of 2 years. Unless there is a specific need for a REST API.

Reply:
On our end, there isn't one.

Reply:
Never mind - its marked internal - this should be fine.